rustler_mix/lib/rustler/compiler/config.ex

```diff
@@ -0,0 +1,83 @@
+defmodule Rustler.Compiler.Config do
+  @moduledoc false
+
+  @type rust_version :: :stable | :beta | :nightly | binary()
+  @type cargo :: :system | {:rustup, rust_version()} | {:bin, Path.t()}
+  @type crate :: atom()
+  @type default_features :: boolean()
+  @type env :: [{binary(), binary()}]
+  @type features :: [binary()]
+  @type mode :: :debug | :release
+  @type load_data :: term()
+  @type path :: Path.t()
+
+  defstruct cargo: :system,
+            crate: nil,
+            default_features: true,
+            env: [],
+            external_resources: [],
+            features: [],
+            load_data: 0,
+            load_from: nil,
+            mode: :release,
+            otp_app: nil,
+            path: "",
+            priv_dir: "",
+            skip_compilation?: false,
+            target: nil,
+            target_dir: ""
+
+  alias Rustler.Compiler.Config
+
+  def from(otp_app, module, opts) do
+    config = Application.get_env(otp_app, module, [])
```

Comment:
Should we document that configuration of a NIF is possible through the application configuration?

Reply:
Perhaps, if this isn't clear enough: https://github.com/rusterlium/rustler/pull/275/files#diff-fe83ab22d5a970ab547c86273d644238R13-R17